assets/_scss/vendor/font-awesome/_icons.scss

```diff
@@ -378,6 +378,7 @@
 .#{$fa-css-prefix}-youtube-play:before { content: $fa-var-youtube-play; }
 .#{$fa-css-prefix}-dropbox:before { content: $fa-var-dropbox; }
 .#{$fa-css-prefix}-stack-overflow:before { content: $fa-var-stack-overflow; }
+.#{$fa-css-prefix}-speaker-deck:before { content: $fa-var-external-link; }
```

Comment:
This is clever, but hacks the library and shouldn't be added to the theme as it it'll break the moment users attempt to upgrade their version of font awesome.